lib/cli.js

```diff
@@ -278,26 +279,36 @@ exports.process = () => {
 }
 
 exports.run = () => {
-  const config = exports.process()
-
-  switch (config.cmd) {
-    case 'start':
-      new Server(config).start()
-      break
-    case 'run':
-      require('./runner')
-        .run(config)
-        .on('progress', printRunnerProgress)
-      break
-    case 'stop':
-      require('./stopper').stop(config)
-      break
-    case 'init':
-      require('./init').init(config)
-      break
-    case 'completion':
-      require('./completion').completion(config)
-      break
+  const cliOptions = exports.process()
+  const cmd = cliOptions.cmd // prevent config from changing the command
+  const cmdSupportsConfigPromise =
```

Comment:
nit: cmdSupportsConfigPromise -> needsConfig

It's not so much about the support, but more like whether this command needs to parse the config file.

Reply:
I'll see your nit pick and raise you one! This seems fine to me, but it would make more sense to me to explicitly say what needs the configuration.

How about `cmdNeedsConfig`?

Reply:
Sounds good!

Consider also swapping if/else branches to avoid inverting the condition.